packages/core/src/modules/event/event-service.ts

```diff
@@ -3,11 +3,12 @@ import { Attendance, AttendanceWrite, Event, EventWrite } from "@dotkomonline/ty
 import { NotFoundError } from "../../errors/errors"
 import { Cursor } from "../../utils/db-utils"
 import { AttendanceRepository } from "./attendance-repository"
+import { EventCommitteeRepository } from "./event-committee-repository"
 import { EventRepository } from "./event-repository"
 
 export interface EventService {
-  createEvent(eventCreate: EventWrite): Promise<Event>
-  updateEvent(id: Event["id"], payload: Omit<EventWrite, "id">): Promise<Event>
+  createEvent(eventCreate: EventWrite, eventCommittees: string[]): Promise<Event>
```

Comment:
Shouldn't include edges on create in my opinion, better to do the setCommittees stuff in the caller.